### ISSUE TITLE
Contribution to the repo and dividing the tasks

### ISSUE THREAD

**Issue by CallmeMehdi** [MLH-Fellowship/Auto-Tagger#1]
Hello Team, I was wondering how we are going to divide the tasks in order to complete the project within Tuesday, we know that James has special conditions, so, we need to re-allocate his part of creating a website to one of the 2 team members to finish the project.
For now, we have completed our base model in a jupyter notebook ( which will be converted to python scripts soon ) so how do we divide the next tasks?

**Comment by pncnmnp** [comment]
I can work with both Pytorch-BentoML integration and our frontend. Can we improve our model? Have you tested the accuracy using a dev set? We should formalize our metrics.

**Comment by CallmeMehdi** [comment]
Yes, I have calculated the loss: Train Loss = 0.07787966687892259 Valid Loss = 0.09229168470948934 .
I think it's a pretty robust model and it works with most names. I will now work on converting my work to seperate scripts to reproduce the results, I will also put the model weights in my drive for easy access ( Because its around 500 MB ).
